projects/addon-doc/components/api/api-item.component.ts

```diff
@@ -81,6 +87,18 @@ export class TuiDocAPIItem<T> implements OnInit {
         this.setQueryParam(value);
     }
 
+    public emitEvent(event: unknown): void {
+        this.emits.update((x) => ++x);
+
+        let content: string | undefined;
```

Comment:
`this.alerts.open(event ?? tuiInspectAny(event, 2), {label: this.name}).subscribe()`

Reply:
Updated